lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -440,7 +440,7 @@ export default class BitcoinProcessor {
     }
 
     // if not enough blocks to fill the page then there are no more transactions
```

Comment:
Sorry, missed this, comment need to be updated?

Reply:
Good catch